Readme.md

```diff
@@ -58,6 +58,21 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 }
 ```
 
+3.a. If you want to be able to change the email address of a user without also changing their user id, enable `StableUserId` mode:
```

Comment:
This should be an addition to the end of the readme (e.g. Modifying RavenDB Conventions), not in the getting started section. Getting started, users can use the defaults.